moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -45,11 +45,21 @@
 #include <set>
 #include <Eigen/Core>
 #include <console_bridge/console.h>
-#include <moveit/robot_model/robot_model.h>
+#include <moveit/macros/class_forward.h>
+
+namespace moveit
+{
+namespace core
+{
+MOVEIT_CLASS_FORWARD(RobotModel)
```

Comment:
The header is only needed to forward-declare some classes. Usually, one avoids including headers in this case, because it causes extra compile overhead.